plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.test.ts

```diff
@@ -106,36 +106,26 @@ describe('fetch:cookiecutter', () => {
       output: jest.fn(),
       createTemporaryDirectory: jest.fn().mockResolvedValue(mockTmpDir),
     };
-
-    // mock the temp directory
-    mockFs({ [mockTmpDir]: {} });
-    mockFs({ [`${join(mockTmpDir, 'template')}`]: {} });
+    mockDir.clear();
+    mockDir.addContent({ template: {} });
```

Comment:
Could this as well as all other usages just be done by `mockDir.setContent(...)`? What is the benefit of `addContent`? :)

Reply:
Let's use setContent!